### DISCUSSION TITLE
Wrong connection type colors after importing project

### DISCUSSION THREAD

**Discussion by michal-bartak** [dbeaver/dbeaver#9811] [Q&A]
It happened to me multiple times. So I devided to report it as a bug.
I sticked to 3 default connection types, assigning them colors as follows:
development - green
test - orange
production - red

Everytime I export project and import onto other machine I got as follows:
development - no color (white)
test - green
production - red (unchanged)

It seems, that colour of dev env is always applied to test one. Maybe it's just array/collection shift while iterating objects. Just guess.

With regards

**Comment by uslss** [comment]
thanks for the bug report

**Comment by LonwoLonwo** [comment]
Hello @michalk-k 

> development - no color (white)
> test - green
> production - red (unchanged)

This are default connection types colours.

When exporting a project to DBeaver on another machine, the connection type colour settings are not saved - the default ones are used.

As a workaround, you can create new connection types and colours and use them. These types must be preserved when exporting the project.

![2020-12-15 15_35_18-Window](https://user-images.githubusercontent.com/45152336/102216212-df541b80-3eeb-11eb-810d-07b65f27460c.png)
